### ISSUE TITLE
Unable to simultaneously hold an overlay button and tap the DS tocuhscreen

### ISSUE THREAD

**Issue by esSteres** [libretro/RetroArch#15749]
## Description

On an android device with the DS pointer mode set to "touch", holding an overlay button disables the DS touchscreen. This is independent of the core being used - I see it happening on melonds and desmume.
I am certain I have multitouch on my phone. It is possible to press two or more overlay buttons at once, and it is also possible to drag or hold on the touchscreen and press an overlay button. It is only the situation where you are first holding an overlay button and then tap the screen that is unusual.

### Expected behavior

It should be possible to hold down a button on the overlay and interact with the touch screen at the same time.

### Actual behavior

Holding an overlay button seems to lock the touchscreen, making it only possible to press other overlay buttons

### Steps to reproduce the bug

It would help to test with a game which relies on this kind of interaction. I discovered it while trying to play Picross 3D for the DS, since that game requires you to hold a d-pad button and tap the screen at the same time.

1. Press and hold any overlay button
2. Tap or drag on the touchscreen area - nothing will happen.

### Bisect Results

Unfortunately, I'm not set up for Android development right now, so I can't do any testing aside from on my phone.

### Version/Commit
 
I merely downloaded the latest version of the RetroArch app from the Play Store. When I check Home > Information > System Information, it reports a git version of 4b16b4f, but I'm unclear on whether this is information about my device or about RetroArch itself. Sorry that I don't have any more specific information.

**Comment by sonninnos** [comment]
Please try with the actual current version first, since that version is years old, and it might already be fixed.

**Comment by esSteres** [comment]
I have tracked down the latest version, and the issue is still present. Also, since I was surprised that the app had not auto-updated, I went back and double-checked; that version (the one from 2021) is the latest one uploaded to the Play Store. According to most people's default way of downloading software, it _is_ the "actual current version". Perhaps you should update your page.

**Comment by sonninnos** [comment]
Yes the Play Store version is ancient, because it can't be updated without conforming to their unrealistic rules.

**Comment by esSteres** [comment]
In any case, as of version 9a57ec2, this is still an issue.
